packages/react/src/ReactContext.js

```diff
@@ -42,6 +42,9 @@ export function createContext<T>(
     // Secondary renderers store their context values on separate fields.
     _currentValue: defaultValue,
     _currentValue2: defaultValue,
+    // Used to track how many concurrent renderers this context currently
+    // supports within in a single renderer. Such as parallel server rendering.
+    _threadCount: 0,
```

Comment:
@trueadm did you copy over the new `react` package? this line is important.

Reply:
Oh. Does this mean `react-dom/server` is not compatible with `react@16.0.0`? We've tried to relax this requirement during 16.x.x release line. If this doesn't work we need to start bumping peer deps again.

Reply:
This was it! I'm sorry, I feel stupid now :(

Reply:
@gaearon We accidentally bumped peer deps anyway.

Reply:
The fix is simple enough I did it anyway. We should relax peer deps back imo.